lib/phoenix/live_dashboard/router.ex

```diff
@@ -380,7 +380,8 @@ defmodule Phoenix.LiveDashboard.Router do
     {pages, requirements} =
       [
         home: {Phoenix.LiveDashboard.HomePage, %{env_keys: env_keys, home_app: home_app}},
-        os_mon: {Phoenix.LiveDashboard.OSMonPage, %{}}
+        os_mon: {Phoenix.LiveDashboard.OSMonPage, %{}},
+        memory_allocators: {Phoenix.LiveDashboard.MemoryAllocatorsPage, %{}}
```

Comment:
I put the new page in the third position, but maybe you want to put it in another position.